[Limbus] Clean, sanitize and enable Apollyon NE and SE

Xaver-DaRed opened the pull request:
<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Changes Global:
- Moved battlefield logic from mob scripts to a single global file.
- Fixed portal at floor 5 in NE and 4 in SE opening, wich shouldn't.
- Fixed oddity with battlefield timer returning max time remaining when entering instead of actual time remaining.
- Removed unused regions and unused NPC IDs.
- Changed player:startEvent to player:startCutscene or player:startOptionalCutscene, when aplicable.

NE
- Fixed Floor 2 never selecting 4th mob as "Portal Opener"

SE
- Fixed treasure chests in floor 3 poping on top of each other.

Notes:
- Non-boss mobs in floors 1 to 4 superlink. They shouldn't.
- Battlefield timer doesn't update when recieving a time extension. (Nevermind the fact this zones don't have a timer in retail.)

Xaver-DaRed committed:
Clean and sanitize Apollyon NE and SE

cocosolos commented:
>Fixed portal at floor 5 in NE and 4 in SE opening, wich shouldn't.
Removed unused regions and unused NPC IDs.

I think I put those final stage portals there after the battlefield is empty as a way for players to get out if they get stuck (logout/crash) since they don't get teleported out like in retail. I'm not sure if they're even set to open in the DB in case of a server crash.

>Fixed treasure chests in floor 2 poping on top of each other.

Is this supposed to be floor 3? I believe they actually can spawn on top of each other, at least according to old wiki. I can't find any videos of this happening though.

Xaver-DaRed commented:
> Cosmo Cleanse timer is supposed to be set when it's used not received.
You are correct. Fixed.

> Is this supposed to be floor 3? I believe they actually can spawn on top of each other, at least according to old wiki. I can't find any videos of this happening though.
Yeah, I tupo'd the floor number in comment. Fixed. I couldn't find any video evidence of it, and it looks weird. That's the reason I made them not do that.
